src/Catalog/PackageCatalogItem.cs

```diff
@@ -94,6 +103,52 @@ public override IGraph CreateContentGraph(CatalogContext context)
             //  identity and version
             SetIdVersionFromGraph(graph);
 
+            // deprecation
+            if (Deprecation != null)
+            {
+                // assert deprecation root node to subject
+                var deprecationPredicate = graph.CreateUriNode(Schema.Predicates.Deprecation);
+                var deprecationRootNode = graph.CreateUriNode(new Uri(resource.Subject.ToString() + "#deprecation"));
+                graph.Assert(resource.Subject, deprecationPredicate, deprecationRootNode);
+
+                // assert reasons to deprecation root node
+                var deprecationReasonRootNode = graph.CreateUriNode(Schema.Predicates.DeprecationReasons);
+                foreach (var reason in Deprecation.Reasons)
+                {
+                    var reasonNode = graph.CreateLiteralNode(reason);
+                    graph.Assert(deprecationRootNode, deprecationReasonRootNode, reasonNode);
+                }
+
+                // assert message to deprecation root node
+                if (Deprecation.Message != null)
+                {
+                    graph.Assert(
+                        deprecationRootNode,
+                        graph.CreateUriNode(Schema.Predicates.DeprecationMessage),
+                        graph.CreateLiteralNode(Deprecation.Message));
+                }
+
+                if (Deprecation.AlternatePackage != null)
```

Comment:
Our DB allows multiple deprecations. Do think it's right to start with it being an array of deprecations but today only ever have one or zero?

Reply:
Good point, however, even though we have multiple deprecations, the idea was always for there to only be a single ACTIVE deprecation.

Having multiple deprecations was always intended to be "this is a deprecation I think you should apply to your package" or "this is a deprecation that our automated security scanning added to your package" and not meant to be actually visible.

Reply:
I see, so we don't foresee users ever seeing multiple deprecation models on one package at the same time.

Reply:
Yes, and I think even if we did, we would be combining them to a single visible deprecation for clients to consume.